test/react/server_rendering/manifest_container_test.rb

```diff
@@ -1,8 +1,6 @@
 require "test_helper"
 
-# Rails 3.x doesn't have `application.assets_manifest=`
-# I don't think we have to support Rails 3 + Sprockets 3 anyways!
-if Rails::VERSION::MAJOR > 3
+if React::ServerRendering::ManifestContainer.compatible?
```

Comment:
I am leaning towards changing this to an explicit check of `Sprockets::Rails::VERSION`. While this variant is less repetitive, I am afraid some unforeseen future change of the `compatible?` method might cause the tests to be skipped for unexpected cases.